features/distro-resources/src/main/resources/items/demo.items

```diff
@@ -26,77 +26,77 @@ Group:Number:AVG                Temperature "Avg. Room Temperature [%.1f °C]"
-Switch Heating_FF_Daughter          "Amelia's Room" <heating>   (FF_Daughter, Heating)
-Switch Heating_FF_Bed               "Bedroom"       <heating>   (FF_Bed, Heating)
+Switch Heating_FF_Bath              "Bath"          <heating>   (FF_Bath, Heating)      ["HVAC"]
+Switch Heating_FF_Office            "Office"        <heating>   (FF_Office, Heating)    ["HVAC"]
+Switch Heating_FF_Son               "Oliver's Room" <heating>   (FF_Son, Heating)       ["HVAC"]
+Switch Heating_FF_Daughter          "Amelia's Room" <heating>   (FF_Daughter, Heating)  ["HVAC"]
+Switch Heating_FF_Bed               "Bedroom"       <heating>   (FF_Bed, Heating)       ["HVAC"]
 
 /* Rollershutters */
-Rollershutter Shutter_GF_Toilet         "Toilet"        (GF_Toilet, Shutters)
-Rollershutter Shutter_GF_Kitchen        "Kitchen"       (GF_Kitchen, Shutters)
-Rollershutter Shutter_GF_Living         "Livingroom"    (GF_Living, Shutters)
+Rollershutter Shutter_GF_Toilet         "Toilet"        (GF_Toilet, Shutters)   ["Blinds"]
+Rollershutter Shutter_GF_Kitchen        "Kitchen"       (GF_Kitchen, Shutters)  ["Blinds"]
+Rollershutter Shutter_GF_Living         "Livingroom"    (GF_Living, Shutters)   ["Blinds"]
 
-Rollershutter Shutter_FF_Bed            "Bedroom"       (FF_Bed, Shutters)
-Rollershutter Shutter_FF_Bath           "Bath"          (FF_Bath, Shutters)
-Rollershutter Shutter_FF_Office_Window  "Office Window" (FF_Office, Shutters)
-Rollershutter Shutter_FF_Office_Door    "Office Door"   (FF_Office, Shutters)
+Rollershutter Shutter_FF_Bed            "Bedroom"       (FF_Bed, Shutters)      ["Blinds"]
+Rollershutter Shutter_FF_Bath           "Bath"          (FF_Bath, Shutters)     ["Blinds"]
+Rollershutter Shutter_FF_Office_Window  "Office Window" (FF_Office, Shutters)   ["Blinds"]
+Rollershutter Shutter_FF_Office_Door    "Office Door"   (FF_Office, Shutters)   ["Blinds"]
 
 Switch Shutter_all
 
 /* Indoor Temperatures */
-Number Temperature_GF_Corridor  "Temperature [%.1f °C]" <temperature>   (Temperature, GF_Corridor)
-Number Temperature_GF_Toilet    "Temperature [%.1f °C]" <temperature>   (Temperature, GF_Toilet)
-Number Temperature_GF_Living    "Temperature [%.1f °C]" <temperature>   (Temperature, GF_Living)
-Number Temperature_GF_Kitchen   "Temperature [%.1f °C]" <temperature>   (Temperature, GF_Kitchen, Thermostat) ["CurrentTemperature"]
-Number Temperature_FF_Bath      "Temperature [%.1f °C]" <temperature>   (Temperature, FF_Bath)
-Number Temperature_FF_Office    "Temperature [%.1f °C]" <temperature>   (Temperature, FF_Office)
-Number Temperature_FF_Son       "Temperature [%.1f °C]" <temperature>   (Temperature, FF_Son)
-Number Temperature_FF_Daughter  "Temperature [%.1f °C]" <temperature>   (Temperature, FF_Daughter)
-Number Temperature_FF_Bed       "Temperature [%.1f °C]" <temperature>   (Temperature, FF_Bed)
+Number Temperature_GF_Corridor  "Corridor [%.1f °C]"        <temperature>   (Temperature, GF_Corridor)  ["Temperature"]
```

Comment:
As we do not have readOnly=true available on these items - should we add "Measurement" here to make clear that it isn't a Control or Setpoint?

Reply:
Yes, good point (almost no pun intended 😁).